Gemfile

```diff
@@ -43,16 +43,23 @@ gem 'travis',         '~> 1.7.6'
 
 gem 'awesome_spawn',        '>= 1.4.1'
 gem 'default_value_for',    '>= 3.1.0'
-gem 'haml_lint',            '~> 0.28.0', :require => false
+gem 'haml',                 '~> 5.1',    :require => false # force newer version of haml
```

Comment:
Why do we want to force it? We're running on `hamlit` which isn't compatible with v5. cc@himdel

Reply:
@skateman Please comment on this in the appropriate PR:

https://github.com/ManageIQ/miq_bot/pull/498

This commit is pulled over from that.  Resolving.